python/llm/example/transformers/transformers_int4/whisper/readme.md

```diff
@@ -27,6 +27,17 @@ Arguments info:
 - `--repo-id-or-data-path REPO_ID_OR_DATA_PATH`: argument defining the huggingface repo id for the audio dataset to be downloaded, or the path to the huggingface dataset folder. It is default to be `'hf-internal-testing/librispeech_asr_dummy'`.
```

Comment:
I suggest adding a new section for long segment recognition, such as the example readme here: https://github.com/intel-analytics/BigDL/blob/main/python/llm/example/transformers/transformers_int4/chatglm2/streamchat.py